### ISSUE TITLE
Convert bag metadata datetime fields to date fields

### ISSUE THREAD

**Issue by helrond** [RockefellerArchiveCenter/aurora#506]
## Is your feature request related to a problem? Please describe.
Bag metadata date fields use Django datetime fields. This is confusing for end users.

## Describe the solution you'd like
Convert datetime fields to date fields.

## Additional context
Will need to investigate how this impacts existing data.

**Comment by McDaPick** [comment]
Converted the DateTime fields to Date fields in the BagInfoMetadata model. 

After a refresh and taking a look at the metadata section in the transfers detail view, it seems bagging_date was pushed forward five hours while date_start and date_end were pushed four. 

Before field change:

![DateTimeStart](https://user-images.githubusercontent.com/22798316/149842257-405f3c62-74c3-42c4-9fe7-c7f810caacfc.PNG)

After field change:

![DateField](https://user-images.githubusercontent.com/22798316/149842271-ff5eb16c-15d9-49ce-8291-550e9fd18ea8.PNG)

Going to grab a burrito 🌯 but will be looking through bag_transfers to see what could be causing this afterwards.

**Comment by helrond** [comment]
@McDaPick Hey, thanks for working on this! I'm guessing there is some timezone/daylight savings time stuff going on. I would suggest the following:
- Look at the underlying values for these Transfers in the Django shell to confirm they are correct. What you see in the browser is getting filtered through templates, which also need to be updated.
- Some work may need to be done in the TransferRoutine (which is where these values get saved) to save dates rather than datetimes. I think this *might* be where you'd find what's going on with bagging_date.

**Comment by McDaPick** [comment]
You would not believe how long that burrito line was! 

Jumped back into this and believe I found the issue above. 

After updating the date_start and date_end fields in the BagInfoMetadata model from 

```
date_start = models.DateTimeField()
date_end = models.DateTimeField()
```

to

```
date_start = models.DateField()
date_end = models.DateField()
```

and running a migration, the date was being saved as one day earlier than what was in the bag-info file. 

**Transfer's Bag Info:** 

External Identifier: records-2017-12-11T20:09:48.238148
BagIt-Profile-Identifier: http://localhost:8000/api/bagit_profiles/2/
Bagging-Date: 2017-12-11T20:09:48.238148
Date-End: 2005-06-22
Date-Start: 2003-05-14

**The Transfer's data through the Django shell after being imported:**

```
>>> from bag_transfer.models import BagInfoMetadata
>>> x = BagInfoMetadata.objects.get(external_identifier="records-2017-12-11T20:09:48.238148")
>>> print(x.date_end)
2005-06-21
>>> print(x.date_start)
2003-05-13
```

Inside of the Transfer's detail view:

![image](https://github.com/RockefellerArchiveCenter/aurora/assets/22798316/49711dc0-b015-4a08-b5fb-2b180fa9c473)

After reading @helrond comment about timezone/daylight, I started to take a look around and saw that `config.py` has the time zone set as America/New_York on line 15.

```
DJANGO_TIME_ZONE = "America/New_York"  # TZ database name for local timezone (string)
```

Updating this value to UTC allowed for the `date_field `to save correctly.

```
DJANGO_TIME_ZONE = "UTC"  # TZ database name for local timezone (string)
```

**Transfer's Bag Info:** 

External Identifier: records-2017-12-11T20:09:48.238148
BagIt-Profile-Identifier: http://localhost:8000/api/bagit_profiles/2/
Bagging-Date: 2017-12-11T20:09:48.238148
Date-End: 2005-06-22
Date-Start: 2003-05-14

**The Transfer's data through the Django shell after `DJANGO_TIME_ZONE` was updated:**

```
>>> from bag_transfer.models import BagInfoMetadata
>>> x = BagInfoMetadata.objects.get(external_identifier="records-2017-12-11T20:09:48.238148")
>>> print(x.date_end)
2005-06-22
>>> print(x.date_start)
2003-05-14
```

Inside of the Transfer's detail view:

![image](https://github.com/RockefellerArchiveCenter/aurora/assets/22798316/2c331d54-4888-4164-bbe0-4f36ed4a82a6)

Side Note: 

This update to the `DJANGO_TIME_ZONE` also seems to fix the `bagging_date` time being fives hours off of its value in bag-data.

I pushed up a branch and submitted a PR with the changes. 

Pull Request #634
